processing/data_collection/gazette/spiders/rj_sao_goncalo.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+
+from gazette.spiders.base import BaseGazetteSpider
+from gazette.items import Gazette
+
+
+class RjSaoGoncaloSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3304904"
+    DOWNLOAD_GAZETTE_URL = "https://servicos.pmsg.rj.gov.br/diario/{}.pdf"
+    start_date = datetime.date(1998, 2, 3)
+
+    name = "rj_sao_goncalo"
+    allowed_domains = ["servicos.pmsg.rj.gov.br"]
+    start_urls = ["https://servicos.pmsg.rj.gov.br/diario_oficial.php"]
+
+    def parse(self, response):
+        date = datetime.date.today()
+        start_date = self.start_date
+        while date >= start_date:
+            is_weekend = date.weekday() >= 5
+            if is_weekend:
```

Comment:
**issue (blocking)** We **do** have gazettes on weekends. I ran this spider replacing this line with `if not weekend` just to get items only for weekends. The majority doesn't have a gazette, however I found 527 gazettes on weekends. For example 11/03/2000 is a Saturday and we have a gazette.

Nevertheless I found something interesting there. We can get the gazettes straight from the generated URLs, but if we search for this date in the website, we don't have any response. It is a bug on their site.

Reply:
You're right, I didn't test this for the entire timespan. It's good that you caught it, thanks ;)